apps/web-giddh/src/app/invoice/invoice.component.ts

```diff
@@ -40,7 +50,15 @@ export class InvoiceComponent implements OnInit, OnDestroy {
 
                 if (params) {
                     if (params.voucherType) {
-                        this.selectedVoucherType = params.voucherType;
+
+                        // this.selectedVoucherType = VoucherTypeEnum[tab];
+                        if (!(params.voucherType === 'pending' || params.voucherType === 'settings')) {
```

Comment:
please test it local
ctrl + credit note
then go to template 
Then you will get issue

Reply:
I have not checked the template. 
The template route can also be fixed by same Jugad. 
But in the future, we should do this cleanup.